Tests/NetworkServiceTests/NetworkErrorTests/NetworkError+LocalizedErrorTests.swift

```diff
@@ -2,60 +2,26 @@ import NetworkService
+    func testLocalizedDescriptionReturnExpectedString() {
         // Given
-        let expectedLocalizedDescription = "The server cannot process the request due to an error in the request."
-        let actualLocalizedDescription = NetworkError.badRequest.localizedDescription
-
-        // When // Then
-        XCTAssertEqual(expectedLocalizedDescription, actualLocalizedDescription)
-    }
-
-    func testLocalizedDescriptionReturnsCorrespondingMessageForAuthenticationError() {
-        // Given
-        let expectedLocalizedDescription = """
+        let badRequestDescription = "The server cannot process the request due to an error in the request."
+        let badResponseDescription = "The response is invalid or couldn't be parsed."
+        let authenticationErrorDescription = """
         The server cannot complete the request.
         It lacks valid authentication credentials for the requested resource.
         """
-        let actualLocalizedDescription = NetworkError.authenticationError.localizedDescription
-
-        // When // Then
-        XCTAssertEqual(expectedLocalizedDescription, actualLocalizedDescription)
-    }
-
-    func testLocalizedDescriptionReturnsCorrespondingMessageForClientError() {
-        // Given
-        let expectedLocalizedDescription = "The server cannot process the request due to a client error."
-        let actualLocalizedDescription = NetworkError.clientError.localizedDescription
-
-        // When // Then
-        XCTAssertEqual(expectedLocalizedDescription, actualLocalizedDescription)
-    }
-
-    func testLocalizedDescriptionReturnsCorrespondingMessageForServerError() {
-        // Given
-        let expectedLocalizedDescription = "The server cannot process the request due to a issue with the server."
-        let actualLocalizedDescription = NetworkError.serverError.localizedDescription
-
-        // When // Then
-        XCTAssertEqual(expectedLocalizedDescription, actualLocalizedDescription)
-    }
-
-    func testLocalizedDescriptionReturnsCorrespondingMessageForRequestTimeoutError() {
-        // Given
-        let expectedLocalizedDescription = "The server timed out waiting for the request."
-        let actualLocalizedDescription = NetworkError.requestTimeout.localizedDescription
-
-        // When // Then
-        XCTAssertEqual(expectedLocalizedDescription, actualLocalizedDescription)
-    }
-
-    func testLocalizedDescriptionReturnsCorrespondingMessageForUnknownError() {
-        // Given
-        let expectedLocalizedDescription = "Unknown error."
-        let actualLocalizedDescription = NetworkError.unknownError.localizedDescription
+        let clientErrorDescription = "The server cannot process the request due to a client error."
+        let serverErrorDescription = "The server cannot process the request due to a issue with the server."
+        let requestTimeoutDescription = "The server timed out waiting for the request." // TODO: response?
```

Comment:
Why do we need TODO here?

Reply:
Forgot to update the wrong error's description. Good catch!